zwave_js/rootfs/etc/services.d/zwave_js/run

```diff
@@ -12,5 +12,8 @@ fi
 # Send out discovery information to Home Assistant
 ./discovery &
 
-# Run daemon
-exec zwave-server ${SERIAL_DEVICE} --config /etc/zwave_config.json
+# Create external config dir so configs can be persisted
+mkdir -p /data/db
```

Comment:
Is this the correct file for creating directories? We have a structure script that creates the data cache.

Reply:
Ah, missed that. Will adjust